src/sentry/api/endpoints/organization_details.py

```diff
@@ -116,6 +122,29 @@ def update_organization_scenario(runner):
         )
 
 
+class TrustedRelaySerializer(serializers.Serializer):
+    def to_representation(self, instance):
+        return convert_dict_key_case(instance, snake_to_camel_case)
+
+    def to_internal_value(self, data):
+        try:
+            ret_val = {
+                u"public_key": data.get(u"publicKey"),
+                u"name": data.get(u"name"),
+                u"description": data.get(u"description"),
+            }
+
+            return ret_val
+        except Exception:
+            return {}
+
+    # Note: 'created' and 'lastModified' are only returned by the API,
+    # their values are controlled by the server and are never consumed by the api as inputs
+    publicKey = serializers.CharField(required=True, allow_null=False, allow_blank=False)
+    name = serializers.CharField(required=False, allow_null=True, allow_blank=True)
```

Comment:
I believe name should be required

Reply:
yes, fields have been removed (serialization not using the fields, done via `to_internal_value()`)